src/docs/feature-flags/index.mdx

```diff
@@ -39,6 +44,16 @@ Features can be scoped by organization, and projects. If you're not confident
 you want a project feature, create an organization level one. In this example
 we'll build a feature called `test-feature` scoped at the _organization_ level.
 
+### Determine the permanency of your feature
+
+Typically we use feature flags for development. They are usually [intended to
+be graduated](#after-launch-graduation). However if you have a feature flag
+that controls a subscription-based feature for sentry.io, your feature flag may
+need to live permanently.
```

Comment:
I found this paragraph to be a little confusing.  Maybe it should be rephrased to be authoritative?  "Flags which are gated by subsctiptions are permanent. All others are temporary."  Obviously the phrasing could be improved but an authoritative tone I think would make this section more clear (and, hopefully, compel good behavior).